libraries/AP_Motors/AP_MotorsMulticopter.cpp

```diff
@@ -823,5 +823,10 @@ bool AP_MotorsMulticopter::arming_checks(size_t buflen, char *buffer) const
         }
     }
 
+    if (_spin_min > 0.5) {
```

Comment:
I think we could go a fair bit lower, I don't recall ever seeing it above 25%. Most likely anyone with it higher should increase PWM min.

Reply:
I agree it could go lower.  I was thinking 0.4 or even 0.3 which is the "High" value listed in the parameter description.

// @Values: 0.0:Low, 0.15:Default, 0.3:High